metabase/metabase
# Add a new prometheus metrics in m.qp.middleware.metrics/adjust

### Description

Context: https://metaboat.slack.com/archives/C0645JP1W81/p1728471378145769

Add a new prometheus counter to count errors in `metabase.query-processor.middleware.metrics/adjust`.

### How to verify

* Set `MB_PROMETHEUS_SERVER_PORT` and then check the `/metrics` endpoint, should see logs like

```
# HELP metabase_query_processor_metrics_total Number metrics processed by the query processor.
# TYPE metabase_query_processor_metrics_total counter
metabase_query_processor_metrics_total 0.0
# HELP metabase_query_processor_metrics_errors_total Number of errors when processing metrics.
# TYPE metabase_query_processor_metrics_errors_total counter
metabase_query_processor_metrics_errors_total 0.0
# HELP metabase_query_processor_metrics_created Number metrics processed by the query processor.
# TYPE metabase_query_processor_metrics_created gauge
metabase_query_processor_metrics_created 1.728947783982E9
# HELP metabase_query_processor_metrics_errors_created Number of errors when processing metrics.
# TYPE metabase_query_processor_metrics_errors_created gauge
metabase_query_processor_metrics_errors_created 1.728947783982E9
```

* Or see [dpsutton's comment](https://github.com/metabase/metabase/pull/48547#issuecomment-2405875446) about testing from the repl
* See included tests

### Checklist
